packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts

```diff
@@ -183,7 +188,7 @@ export class ApexLibraryTestRunExecutor extends LibraryCommandletExecutor<
 
   constructor() {
     super(
-      nls.localize('apex_test_run_text'),
+      nls.localize('force_apex_test_run_text'),
       'force_apex_execute_library',
```

Comment:
let's change the log name parameter here to be something like `force_apex_test_run_library`

Reply:
lol whoops good catch